tools/test_cs_write_and_validate.py

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env python3
+
+from util.cassandra_stress import *
+from util.scylla_docker import *
+
+from cassandra.cluster import Cluster
+import argparse
+
+### This test populates the DB with cassandra-stress,
+### validates the contents with cql-stress-cassandra-stress,
+### and vice-versa.
+def run(args, node: ScyllaDockerNode, cs: CassandraStress):
+    cluster = Cluster([node.ip], port=node.port)
+    session = cluster.connect()
+
+    # All setup, start the test.
+    print("\n=== Starting the test... ===")
+
+    # Populate DB with cassandra-stress.
+    print("\n=== Populating database with cassandra-stress... ===\n")
+    cs.run(command="write",
+           workload_size=args.workload_size,
+           node_ip=node.ip, keyspace="ks_cassandra",
```

Comment:
`ks_cassandra` and `ks_cqlstress` should be variables. Furthermore, I think we should generate unique names for those keyspaces and avoid dropping at the end of the test - see my other comments.